### DISCUSSION TITLE
Carbon capture & storage data hack - #t21-hack-ccs

### DISCUSSION THREAD

**Discussion by kwinkunks** [softwareunderground/transform-2021-hackathon#8] [Projects]
A hackathon project on creating a benchmark dataset on a major CCS project.

## Slack channel

`#t21-hack-ccs`

## Goal

The goal is to create one or more benchmark datasets from open data on [the Northern Lights CCS dataset](https://data.equinor.com/dataset/NorthernLights). This would include defining a clear data science challenge on the benchmark.

## People

- Possibly Chance and Luisa

## Resources

- I made [a page on the wiki](https://subsurfwiki.org/wiki/Northen_Lights_dataset) with more info about the dataset.
- [Here's what I think makes a good benchmark dataset.](https://agilescientific.com/blog/2019/4/3/what-makes-a-good-benchmark-dataset)

**Comment by filippo82** [comment]
[T21 Hackathon Idea Mashup](https://softwareunderground.org/events/2021/4/7/t21-hackathon-idea-mashup) today (or tomorrow depending on your time zone) Wednesday April 14th at 16:00 UTC. Come to Slack to find the link for the meeting!
